apache/couchdb-fauxton
# Sidebar redux refactoring

## Overview

Refactoring of sidebar components to use Redux

## Testing recommendations

- Click the sidebar options, selected items should be highlighted
- Expand/collapse design docs subitems
- Create new view. Sidebar should be updated to include the new design doc, and the new view is expanded


## Checklist
